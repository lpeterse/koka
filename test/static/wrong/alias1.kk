import static/modA
import static/modAWrong

alias myint = aint
