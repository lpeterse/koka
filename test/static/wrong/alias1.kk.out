test/static/wrong/alias1.kk(4,15): error: Type aint is ambiguous.
 hint: It can refer to either static/modAWrong/aint, or static/modA/aint